dts/bindings/sent/sent-controller.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Single Edge Nibble Transmission (SENT) Controller
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    Each child node defines the configuration of a Single Edge Nibble Transmission (SENT) channel
+
+  properties:
+    reg:
+      type: int
+      required: true
+      description: Channel identifier.
+
+    num-data-nibbles:
+      type: int
+      enum:
+        - 1
+        - 2
+        - 3
+        - 4
+        - 5
+        - 6
+        - 7
+        - 8
+      required: true
+      description: |
+        Specifies the number of data nibbles. A nibble is a four-bit aggregation.
+        This property defines how many such nibbles are used in the data transmission.
+
+    clock-tick-length-us:
```

Comment:
Same question- if both of these channel properties are set by the sender and static, than I think it makes sense to keep them in devicetree. Otherwise should there be an API to set these?

Reply:
as my reply [above](https://github.com/zephyrproject-rtos/zephyr/pull/80117#discussion_r2165476748)